go/ms/internal/sqlite/db.go

```diff
@@ -0,0 +1,228 @@
+//to return a slice of FullMapRow objects
+func (e *executor) GetFullMap(ctx context.Context) ([]FullMapRow, error) {
+	e.RLock()
+	defer e.RUnlock()
+	rows, err := e.db.QueryContext(ctx, FullMap)
+	if err != nil {
+		return nil, serrors.Wrap(db.ErrReadFailed, err)
+	}
+	defer rows.Close()
+	got := []FullMapRow{}
+	for rows.Next() {
+		var r FullMapRow
+		err = rows.Scan(&r.ID, &r.IP, &r.IA, &r.Timestamp)
+		if err != nil {
+			return nil, serrors.Wrap(db.ErrDataInvalid, err)
+		}
+		got = append(got, r)
+	}
+	return got, nil
+}
+
+//GetFullMapEntryByIP queries the full_map table by IP
+func (e *executor) GetFullMapEntryByIP(ctx context.Context, ip string) ([]FullMapRow, error) {
+	e.RLock()
+	defer e.RUnlock()
+	rows, err := e.db.QueryContext(ctx, FullMapEntryByIP, ip)
+	if err != nil {
+		return nil, serrors.Wrap(db.ErrReadFailed, err)
+	}
+	defer rows.Close()
+	got := []FullMapRow{}
+	for rows.Next() {
+		var r FullMapRow
+		err = rows.Scan(&r.ID, &r.IP, &r.IA, &r.Timestamp)
+		if err != nil {
+			return nil, serrors.Wrap(db.ErrDataInvalid, err)
+		}
+		got = append(got, r)
+	}
+	return got, nil
+}
+
+//InsertFullMapEntry inserts a row into the full_map table
+func (e *executor) InsertFullMapEntry(ctx context.Context, fmRow FullMapRow) (sql.Result, error) {
+	//TODO (supraja): handle transaction correctly here
+	res, err := e.db.ExecContext(ctx, InsFullMapEntry, fmRow.IP, fmRow.IA, fmRow.Timestamp)
+	if err != nil {
+		return nil, err
+	}
+	return res, nil
+}
+
+//DeleteFullMapEntryByID deletes a row in the full_map table by id
+func (e *executor) DeleteFullMapEntryByID(ctx context.Context, id int) (sql.Result, error) {
+
+	//TODO (supraja): handle transaction correctly here
+	res, err := e.db.ExecContext(ctx, DelFullMapEntry, id)
+	if err != nil {
+		return nil, err
+	}
+	return res, nil
+}
+
+//InsertNewEntry inserts a row into the new_entries table
+func (e *executor) InsertNewEntry(ctx context.Context, entry []byte) (sql.Result, error) {
+
+	//TODO (supraja): handle transaction correctly here
+	res, err := e.db.ExecContext(ctx, InsNewEntry, entry)
+	if err != nil {
+		return nil, err
+	}
+	return res, nil
+}
+
+//InsertPCNRep inserts a row into the pcn_reps table
+func (e *executor) InsertPCNRep(ctx context.Context, entry []byte) (sql.Result, error) {
+
+	//TODO (supraja): handle transaction correctly here
+	res, err := e.db.ExecContext(ctx, InsPCNRep, entry)
+	if err != nil {
+		return nil, err
+	}
+	return res, nil
+}
+
+//GetNewEntryByID queries the new_entries table by id to return the signedPld stored in it
+func (e *executor) GetNewEntryByID(ctx context.Context, id int) (*ctrl.SignedPld, error) {
+	e.RLock()
+	defer e.RUnlock()
+	rows, err := e.db.QueryContext(ctx, NewEntryByID, id)
+	if err != nil {
+		return nil, serrors.Wrap(db.ErrReadFailed, err)
+	}
+	defer rows.Close()
+
+	got := &ctrl.SignedPld{}
+
+	rawResult, err := getRawResultFromRows(rows)
+
+	if err != nil {
```

Comment:
```suggestion
	rawResult, err := getRawResultFromRows(rows)
	if err != nil {
```
You can remove the newline between the function call and the error check.